integration_test/Dialect/Handshake/conditional_modification/conditional_modification.mlir

```diff
@@ -2,11 +2,7 @@
 
 // This test is executed with all different buffering strategies
 
-// RUN: circt-opt %s --lower-std-to-handshake \
-// RUN:   --canonicalize='top-down=true region-simplify=true' \
-// RUN:   --handshake-materialize-forks-sinks --canonicalize \
-// RUN:   --handshake-insert-buffers=strategy=all --lower-handshake-to-firrtl | \
-// RUN: firtool --format=mlir --verilog --lowering-options=disallowLocalVariables > %t.sv && \
+// RUN: hlstool %s --dynamic --ir-input-level 0 --verilog --lowering-options=disallowLocalVariables > %t.sv && \
```

Comment:
NIT: We could now drop the `--ir-input-level 0` here and in most of the other tests